pkg/controller/keystone/keystone_controller.go

```diff
@@ -383,12 +332,8 @@ func newKeystoneSTS(cr *contrail.Keystone) *apps.StatefulSet {
 							Args:            []string{"-c", initDBScript},
 							Env: []core.EnvVar{
 								{
-									Name: "MY_POD_IP",
-									ValueFrom: &core.EnvVarSource{
-										FieldRef: &core.ObjectFieldSelector{
-											FieldPath: "status.podIP",
-										},
-									},
+									Name:  "MY_DB_IP",
```

Comment:
Let's name it maybe PSQL_ENDPOINT or PSQL_IP

Reply:
PSQL_IP pls since endpoint sounds like sth more than ip